extra_tests/snippets/syntax_annotations.py

```diff
@@ -0,0 +1,10 @@
+from typing import get_type_hints
+
+def func(s: str) -> int:
+    return int(s)
+
+hints = get_type_hints(func)
+
+# The order of type hints matters for certain functions
+# e.g. functools.singledispatch
+assert hints == {'s': str, 'return': int}
```

Comment:
This test doesn't check the order. It have to be serialized to check the order.

```suggestion
assert list(hints.items()) == [('s', str), ('return': int)]
```